configure.ml

```diff
@@ -1069,6 +1069,12 @@ let write_dbg_wrapper f =
   Unix.chmod f 0o555
 
 let _ = write_dbg_wrapper "dev/ocamldebug-coq"
+(* next line gets Exception: Unix.Unix_error (Unix.ENOENT, "execv", "/usr/bin/echo").
+let _ = Unix.execv "/usr/bin/echo" [|"HI MOM"|] *)
+(* todo: need to remove \r chars from ocamldebug-coq so it will run on Windows
```

Comment:
Is it `ocamldebug-coq` or `ocamldebug-coq.run` that has problematic \r characters? The former we generate at the line above so we should be able to avoid the issue at that time.

Reply:
The problem is in ocamldebug-coq:

```
$ dev/ocamldebug-coq bin/coqtop.byte.exe
dev/ocamldebug-coq: line 2: $'\r': command not found
dev/ocamldebug-coq: line 4: $'\r': command not found
dev/ocamldebug-coq: line 6: $'\r': command not found
dev/ocamldebug-coq: line 10: $'\r': command not found
: No such file or directory: C:\Users\Jim.DESKTOP-MMPNVGH\Documents\coqSource\coq
```

I doubt there is a clean way to avoid generating the \r characters.  The standard I/O libraries in Cygwin convert "\n" to the two characters "\r\n" when writing to text (non-binary) files.  That's so the files are legible when you open them with other Windows tools.